### ISSUE TITLE
Question in torch_utils.py

### ISSUE THREAD

**Issue by VvvvvGH** [ultralytics/yolov5#18]
I found something wired when I am looking at torch_utils.py. 
If you want to reduce randomness, shouldn't do as pytorch document suggested to set `torch.backends.cudnn.deterministic = True` and 
`torch.backends.cudnn.benchmark = False` ?

https://github.com/ultralytics/yolov5/blob/7c2832cd4903e072bd4e20e175144101ff088313/utils/torch_utils.py#L12-L18

**Comment by glenn-jocher** [comment]
@VvvvvGH yes you are correct. The comment is incorrect, the settings are coded for the fastest training.

**Comment by glenn-jocher** [comment]
This should be resolved in https://github.com/ultralytics/yolov5/commit/22d6088205bc2a8d6abfdb9a9a6c0e9a6d7b79f9 now (incorrectly labelled as a #17 fix). 

Default training seed changed to 1 now as part of changes.
